jobfunnel/__main__.py

```diff
@@ -34,19 +45,29 @@ def main():
     jf.update_filterjson()
 
     # get jobs by either scraping jobs or loading dumped pickles
-    if config['recover']:
+    if config["recover"]:
         jf.load_pickles(config)
-    elif config['no_scrape']:
+    elif config["no_scrape"]:
         jf.load_pickle(config)
     else:
-        for p in config['providers']:
-            provider: Union[GlassDoor, Monster, Indeed] = PROVIDERS[p](config)
+        for p in config["providers"]:
```

Comment:
I think this can be removed in lieu of having both the static and dynamic glassdoor as providers